sdk/webpubsub/Azure.Messaging.WebPubSub/src/swagger/WebPubSub.json

```diff
@@ -1,933 +1,1012 @@
 ﻿{
```

Comment:
Can you change your formatting settings so the diff is reviewable?

Reply:
If you hit the settings (gear) icon on the top and enable 'Hide whitespace changes' and reload, you will get a better diff with only the changes.
One change I have made in this file is to match the spacing with the REST spec .json file, so that diff is easier (until we remove this file).